src/test/kotlin/com/ampnet/blockchainapiservice/service/EthereumSignatureCheckerServiceTest.kt

```diff
@@ -0,0 +1,71 @@
+package com.ampnet.blockchainapiservice.service
+
+import com.ampnet.blockchainapiservice.TestBase
+import com.ampnet.blockchainapiservice.util.SignedMessage
+import com.ampnet.blockchainapiservice.util.WalletAddress
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.Test
+
+class EthereumSignatureCheckerServiceTest : TestBase() {
```

Comment:
Could you add a signature using ledger? You know that we had some problems with verifying ledger signature

Reply:
This is handled by that `V_OFFSET` in `EthereumSignatureCheckerService`. But I will add a Ledger-signed value for tests here also.